### PR TITLE
9.0 porting l10n_it_rea

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by archetipo** [comment]
:+1:

File: l10n_it_rea/README.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-italy/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_rea%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_rea%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (tafaRU) [inline l10n_it_rea/README.rst]: Meanwhile, the Bug Tracker section has been changed, see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker as reference.
Could you please modify it?

File: l10n_it_rea/__openerp__.py [hunk 2]
```diff
@@ -19,19 +19,19 @@
...
     'website': 'http://www.agilebg.com',
     'license': 'AGPL-3',
     "depends": [
-        'l10n_it_base', 'account'
+        'account'
     ],
     "data": [
         'partner_view.xml',
     ],
     "test": [],
```
Comment (tafaRU) [inline l10n_it_rea/__openerp__.py]: Please remove empty keys.

**Comment by tafaRU** [comment]
Only some nitpickings otherwise :+1:

**Comment by eLBati** [comment]
@tafaRU done, thanks

**Comment by tafaRU** [comment]
:+1:

**Pull request merged**